test/helloword.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from 'chai';
+import { describe, it } from "mocha";
+import * as fs from 'fs';
+
+describe('Sanity check for WebAssembly', () => {
+    it("should be available", () => {
+        expect(WebAssembly.Module).to.be.not.eq(undefined);
+    });
+});
+
+describe('Basic u128 operations', () => {
+    var file = fs.readFileSync("test/build/u128.wasm");
+    var memory = new WebAssembly.Memory({initial: 1});
+    var buffer = new Uint8Array(memory.buffer);
+    var instance : any;
+
+    var imports = {
+        env: {
+            abort: () => {},
+            memory: memory
+        }
+    };
+
+    WebAssembly.instantiate(file, imports)
+        .then((result) => {
+            instance = result.instance.exports;
+        })
+        .catch((error) => {
+            console.log(error);
+    });;
+
+    it("should add 1 plus 1", () => {
+        let p = instance.plus(1,1);
+        expect(buffer[p]).to.be.eq(2);
+    })
+    it("should add 100 plus 100", () => {
+        let p = instance.plus(100,100);
+        expect(buffer[p]).to.be.eq(200);
+    })
+    it("should add 1000000 plus 1000000", () => {
+        let p = instance.plus(1000000,1000000);
+        for (var i = 0; i < 32; i++) {

```

Comment:
It seems this part great candidate for making general util which check u128 result.